plugins/destination/postgresql/client/types.go

```diff
@@ -4,7 +4,16 @@ import (
 	"github.com/cloudquery/plugin-sdk/schema"
 )
 
-func SchemaTypeToPg(t schema.ValueType) string {
+func (c *Client) SchemaTypeToPg(t schema.ValueType) string {
+	switch c.pgType {
+	case pgTypeCockroachDB:
+		return c.SchemaTypeToCockroach(t)
+	default:
+		return c.SchemaTypeToPg10(t)
```

Comment:
I thought our minimum supported pg version was 11

Reply:
I tested it locally, we should change our Postgresql In CI to 10. there is no feature in 11 right now that we use that is not available in 10